Scripts/post_build.ps1

```diff
@@ -107,6 +107,13 @@ function Publish-Self-Contained ($p) {
     dotnet publish -c Release $csproj /p:PublishProfile=$profile
 }
 
+function Publish-Portable ($outputLocation, $version) {
+    
+    & $outputLocation\Flow-Launcher-v$v.exe --silent | Out-Null
+    mkdir "$env:LocalAppData\FlowLauncher\app-$version\UserData"
+    Compress-Archive -Path $env:LocalAppData\FlowLauncher -DestinationPath $outputLocation\Flow-Launcher-Portable.zip
+}
+
```

Comment:
Why not simply compress the folder that has been published before deleting it?

Reply:
we need to add userdata in there first

Reply:
We may also need to create a default setting.json and mark Portable as true.

Reply:
no, portable is determined simply if UserData folder exists

Reply:
> Why not simply compress the folder that has been published before deleting it?

Also you need to Squirrelfy it first otherwise you dont get the Update.exe, which will cause restart and update functionality to fail.  However Squirrelfy packs everything into an installer so, in the end you just have to install it then convert it to portable.

Reply:
Oh ok, so why not simply create UserData folder and zip it?

Reply:
because that will make the installer portable

Reply:
Sorry for interject 😄 but this is a thing that I request long ago on the old repo, is keep the portable release as a zip file, really simple, no magic.

I think portable users like that more than current approach. The current approach is...a bit new, not many users familiar with that workflow.

You see this approach in nearly all portable software, one zip file, no installer, run independently.

https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Windows
https://github.com/NickeManarin/ScreenToGif/releases/tag/2.33
https://github.com/greenshot/greenshot/releases/tag/Greenshot-RELEASE-1.2.10.6

Reply:
lol I know, I remember 😉  that's why this is here

It just wasnt as straight forward because the way we pack flow into an installer

Reply:
Got it!, just want to make sure the idea is clear, if it's technical problem then I don't have any more comment haha 😆

Reply:
> because that will make the installer portable

I mean after packing the installer.

Reply:
the directory structure is not setup to use it
![image](https://user-images.githubusercontent.com/26427004/126865568-a8079d4e-24e6-4f55-b2ee-6876fc62febf.png)